src/testE2E/index.ts

```diff
@@ -6,5 +6,5 @@
 import { runTests } from '../test/testRunner'
 
 export function run(): Promise<void> {
-    return runTests('src/testE2E', ['src/testInteg/globalSetup.test.ts'])
+    return runTests(process.env.TEST_DIR ?? 'src/testE2E', ['src/testInteg/globalSetup.test.ts'])
```

Comment:
Seems strange that a test job would have tests located in some other directory, but the path `src/testInteg/globalSetup.test.ts` is still resolved correctly.